platform/feature-api/api/manifest-parser/src/tests/__snapshots__/manifest.test.ts.snap

```diff
@@ -1,116 +1,75 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Parsing Failed parse fail-absolute.json 1`] = `
```

Comment:
These are no longer used, and would be better if they were eliminated. That can be done later, however.